examples/models/lstm/model.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+# Copyright 2024 Arm Limited and/or its affiliates.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import logging
+
+import torch
+
+from torch.nn.quantizable.modules import rnn
```

Comment:
Nit: FYI https://github.com/pytorch/pytorch/blob/main/torch/nn/quantized/modules/rnn.py its moving to torch/ao not sure if we should use that one.